samples/node/slack/README.adoc

```diff
@@ -0,0 +1,148 @@
+= slack-command
+
+[horizontal]
+*Language*:: JavaScript
+*Function*:: https://github.com/projectriff/riff/blob/master/samples/slack/slack-command/slack_command.js[slack_command.js]
+*Protocol*:: HTTP
+*Input*:: slack
+*Output*:: none (replies, implicitly)
+
+== Deploy the FaaS
+
+[NOTE]
+This example requires a Kubernetes cluster with LoadBalancer support and a public facing IP address such as https://cloud.google.com/kubernetes-engine/[GKE].
+Alternately, you can use an http tunnel such as https://ngrok.com/[ngrok].
+
+Deploy riff with link:../../../Getting-Started.adoc[helm].
+
+== Running on minikube with ngrok
+
+Once riff is running and ngrok is installed, get the URL of the http-gateway:
+
+```
+export NODE_PORT=$(kubectl get --namespace default -o jsonpath="{.spec.ports[0].nodePort}" services demo-riff-http-gateway)
+export NODE_IP=$(kubectl get nodes --namespace default -o jsonpath="{.items[0].status.addresses[0].address}")
+echo http://$NODE_IP:$NODE_PORT
+```
+You should see something like:
+```
+http://192.168.99.100:32595
+```
+
+In a separate terminal, start ngrok (substituting your actual URL):
+```
+ngrok http 192.168.99.100:32595
+```
+
+You should see some output including something like:
+
+```
+Forwarding                    http://a61d8bb0.ngrok.io -> 192.168.99.100:32595
+```
+
+=== Build the Docker container for the function code
+
+```
+eval $(minikube docker-env)
+docker build -t projectriff/slack-command:test samples/node/slack/slack-command
+```
+=== Create the Function and Topic(s)
+
+```
+./riff apply -f samples/node/slack/slack-command
+```
+
+=== Test the function locally
+
+Since the input topic configured for the function is named `slack`, configure the slash command to `POST` to `http://<ngrokid>.ngrok.io/requests/slack`.
+
+Slack slash commands POST `x-www-form-urlencoded` content with the posted content contained in `text`. To emulate this you can use curl (substituting your actual URL):
+
+```
+curl -H"Content-Type:application/x-www-form-urlencoded" http://<ngrokid>.ngrok.io/requests/slack -d text='riff is for functions'
+received:riff is for functions
+```
+[TIP]
+You can use the local URL `http://$NODE_IP:$NODE_PORT` here, but a publicly accessible URL is required for Slack integration.
+
+
+[slack-local]
+=== Set up your Slack integration
+Use the `ngrok.io` URL To create your slack https://api.slack.com/slash-commands[slash command].
```

Comment:
Actually, I think the section I commented about above should be here.

And maybe put the "simulating locally" section *after* the real (with a link allowing to skip to it just before the "real" section)